.github/workflows/release.yml

```diff
@@ -191,26 +191,28 @@ jobs:
 
           tar -cvzf ./tmp/${{ steps.create-artifact.outputs.artifact_name }}.tgz -C ${{ env.TMP_BUILD_DIR }} .
 
-      #      - name: Publish Windows iota binary to Chocolatey
-      #        if: ${{ matrix.os == 'windows-latest' && github.event_name == 'release' &&  && contains(env.iota_ref, '-rc') }}
-      #        shell: bash
-      #        run: |
-      #          choco install checksum
-      #          export iota_sha=$(checksum -t sha256 ${{ env.TMP_BUILD_DIR }}/iota.exe)
-      #          cd chocolatey
-      #
-      #          cat <<EOF >>VERIFICATION.txt
-      #          IOTA Binary verification steps
-      #          1. Download https://github.com/iotaledger/iota/releases/download/${{ env.iota_ref }}/${{ env.artifact_name }}.tgz
-      #          2. Extract iota.exe
-      #          3. Verify binary: checksum.exe -t sha256 iota.exe: ${iota_sha}
-      #
-      #          File 'LICENSE.txt' is obtained from: https://github.com/iotaledger/iota/blob/develop/LICENSE
-      #          EOF
-      #
-      #          choco pack --version ${{ env.iota_ref }} configuration=release
-      #          choco apikey --api-key ${{ secrets.CHOCO_API_KEY }} --source https://push.chocolatey.org/
-      #          choco push iota.${{ env.iota_ref }}.nupkg --source https://push.chocolatey.org/
+      - name: Publish Windows iota binary to Chocolatey
+        if: ${{ matrix.os == 'windows-latest' && github.event_name == 'release' && contains(github.ref, '-rc') }}
+        shell: bash
+        run: |
+          choco install checksum
+          export iota_sha=$(checksum -t sha256 ${{ env.TMP_BUILD_DIR }}/iota.exe)
+          cd chocolatey
+
+          cat <<EOF >>VERIFICATION.txt
+          IOTA Binary verification steps
+          1. Download https://github.com/iotaledger/iota/releases/download/${{ needs.generate-tag.outputs.iota_ref }}/${{ env.artifact_name }}.tgz
+          2. Extract iota.exe
+          3. Verify binary: checksum.exe -t sha256 iota.exe: ${iota_sha}
+          4. Verify file 'LICENSE.txt': https://github.com/iotaledger/iota/blob/develop/LICENSE
+          EOF
+
+          # Strip pre-release as we want to publish rc as release version on choco
+          export choco_version=$(echo "${{ needs.generate-tag.outputs.iota_ref }}" | sed 's/^v//' | sed 's/-.*$//')
```

Comment:
And why exactly do we do that? So we release "release candidates" under the wrong tag as "release version"?

Reply:
In choco we could also do pre releases. So release whatever is stable which can be installed with `choco install iota`
and pre-releases as `choco install iota --pre`. I just wanted to release the same version as we do with homebrew.
If we keep the pre-release tag in the version choco won't let you install with `choco install iota` 🤔 Seems a little weird if we keep it like that. But not sure maybe I'm missing something. This is my first choco release